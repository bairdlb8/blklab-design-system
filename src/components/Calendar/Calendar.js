import React from 'react'
import PropTypes from 'prop-types'

import './Calendar.scss'

const Calendar = ({ className = '' }) => {
  return <div>Calendar</div>
}

Calendar.propTypes = {}

export default Calendar
