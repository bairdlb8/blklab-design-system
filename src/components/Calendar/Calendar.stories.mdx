import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Calendar from './Calendar'

<Meta title="Components / Calendar" component={ Calendar } />

# Calendar

## Add some text here

export const Template = (args) => {
  return <Calendar {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Calendar } />
