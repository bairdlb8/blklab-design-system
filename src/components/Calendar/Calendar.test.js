import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { toHaveNoViolations } from 'jest-axe'
import Calendar from './Calendar'

expect.extend(toHaveNoViolations)

describe('<Calendar />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Calendar />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Calendar />)

    expect(asFragment(<Calendar />)).toMatchSnapshot()
  })

})
