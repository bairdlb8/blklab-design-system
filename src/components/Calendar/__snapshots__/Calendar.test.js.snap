// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calendar /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Calendar
  </div>
</DocumentFragment>
`;
